src/app/actions/swap.js

```diff
@@ -21,6 +22,7 @@ export const swapError = createAction('ERROR', (id, error, errorType = '') => ({
 export const swapOrderStatusUpdated = createAction('STATUS_UPDATED', (id, status) => ({ id, orderStatus: status }))
 export const swapTxIdUpdated = createAction('TX_ID_UPDATED', (id, txId) => ({ id, txId }))
 
+export const addManualProperty = createAction('ADD_MANUAL_PROPERTY', idPayload)
```

Comment:
You don't need to store `isManual` in the state. You can add it as a field in the swap extender using the following condition `isManual: !swap.sendWalletId`